taf/updater/updater.py

```diff
@@ -514,6 +519,8 @@ def _merge_branch_commits(
 
 
 def _merge_commit(repository, branch, commit_to_merge, allow_unauthenticated=False):
+    """ Merged the specified branch into the given branch
```

Comment:
Merge~d~ the specified **commit** into the given branch? what does allow_unauthenticated mean? is that the same as error-if-unauthenticated?

Reply:
`allow_unatuhenticated` indicates if a repository can contain authenticated commits and is specified in the authenticated repository. I think that it will be necessary to put more effort into figuring out what exactly to check out at the end of update process. At the moment, repositories where unauthenticated commits are allowed and where they are not are handled differently - but this decision was based on my analysis of our needs more so than anything else.